build/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:18.04
+FROM registry.access.redhat.com/ubi7-dev-preview/ubi-minimal:7.6
```

Comment:
seems good...but has this been considered for all requirements ?

Reply:
yes!! with ubuntu image signing certificate issue is comming